va/meson.build

```diff
@@ -256,3 +268,16 @@ if WITH_WAYLAND
     include_directories : configinc,
     dependencies : deps)
 endif
+
+if meson.is_subproject()
+  python3 = import('python').find_installation()
+  copy_header = '''
+import os, shutil, sys
+outdir = os.path.join(os.environ["MESON_BUILD_ROOT"], os.environ["MESON_SUBDIR"])
+shutil.copy(sys.argv[1], outdir)
+'''
+
+  foreach header : libva_headers_subproject
+    run_command(python3, '-c', copy_header, header, check: true)
+  endforeach
```

Comment:
Can't you just do something like this (with more path/subdir shenanigans if needed)?
```meson
foreach header : libva_headers_subproject
  configure_file(output: header, input: header, copy: true)
endforeach
```

Reply:
done! thanks!